src/components/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string;
+};
+
+type State = {
+  today: Date;
+};
+
+export class Clock extends Component<Props, State> {
+  state: State = {
+    today: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const { clockName } = this.props;
+
+    if (prevProps.clockName !== clockName) {
+      // eslint-disable-next-line no-console
+      console.debug(`Renamed from ${prevProps.clockName} to ${clockName}`);
+    }
+  }
+
+  componentWillUnmount() {
+    window.clearInterval(this.timerId);
+  }
+
+  render() {
+    const { clockName } = this.props;
+    const { today } = this.state;
+    const currentTime = today.toUTCString().slice(-12, -4);
+
+    return (
+      <div className="Clock">
+        <strong className="Clock__name">
+          {clockName}
+        </strong>
+
+        {' time is '}
```

Comment:
```suggestion
        </strong>
        	time is
```

Reply:
I left it like this because of the spaces before and after 
![Screenshot_57](https://user-images.githubusercontent.com/110488158/203292359-e3bb1310-f650-4a47-8035-5056fa086dc4.png)
![Screenshot_56](https://user-images.githubusercontent.com/110488158/203292363-32d50806-74c7-4c69-a0a1-d22b82712cbe.png)